src/main/java/uk/ac/cam/cl/dtg/segue/api/AbstractSegueFacade.java

```diff
@@ -230,4 +230,20 @@ public static boolean isUserStaff(final UserAccountManager userManager, final Ht
         return userManager.checkUserRole(request,
                 Arrays.asList(Role.ADMIN, Role.STAFF, Role.CONTENT_EDITOR, Role.EVENT_MANAGER));
     }
+
+    /**
+     * Is the current user in an admin or content editor role.
+     *
+     * @param userManager
+     *            - Instance of User Manager
+     * @param request
+     *            - with session information
+     * @return true if user is logged in as an admin or content editor, false otherwise.
+     * @throws NoUserLoggedInException
+     *             - if we are unable to tell because they are not logged in.
+     */
+    public static boolean isUserAnAdminOrContentEditor(final UserAccountManager userManager,
+                                                      final HttpServletRequest request) throws NoUserLoggedInException {
+        return userManager.checkUserRole(request, Arrays.asList(Role.ADMIN, Role.CONTENT_EDITOR));
```

Comment:
I think there's an `isUserStaff(...)` at the facade level, too?

Reply:
Ah I didn't realise that was the right access level; changed it.